docs/index.bs

```diff
@@ -2557,6 +2565,12 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
         Note: Keep this definition in sync with [=fetch a classic worker-imported script=].
 
+  A <dfn id="dfn-race-result">race result</dfn> is a tuple of a [=/race response=] and [=/used route=].
```

Comment:
Sorry for not noticing this in the past look, but it might be good to use [=tuple=] to point https://infra.spec.whatwg.org/#tuple.